Measurements/Measurements.ipynb

```diff
@@ -32,14 +32,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%package Microsoft.Quantum.Katas::0.12.20082513"
+    "%package"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "> The package versions in the output of the cell above should always match. If you are running the Notebooks locally and the versions do not match, please install the IQ# version that matches the version of the `Microsoft.Quantum.Katas` package.\n",
+    "> The package versions in the output of the cell above should always match the latest version which is `Microsoft.Quantum.Katas::0.12.20082513`. If you are running the Notebooks locally and the versions do not match, please install the IQ# version that matches the version of the `Microsoft.Quantum.Katas` package.\n",
     "> <details>\n",
     "> <summary><u>How to install the right IQ# version</u></summary>\n",
```

Comment:
This is unrelated to your change, but I just noticed that these instructions are not correct for users who have installed IQ# using Anaconda. For those users, the command would be just one line:
```
conda update -c quantum-engineering qsharp==0.12.20082513
```
@tcNickolas do you think it is worth updating all of the notebooks to include this instruction as well, since `conda` is now the recommended installation path for IQ#? I would be happy to do that in a future PR.

Reply:
I think it could not hurt to add this.

Reply:
I really liked the part of your PR in which the installation instructions were removed from the notebooks altogether together with the `%package` and `%workspace reload` cells. Ideally there should be one place for installation instructions, and that's the main project README; I don't really like the idea of covering all possible scenarios of installation in each notebook.

Reply:
@tcNickolas yes, I like that part of my PR too. 😊

If you think it's appropriate, then, I like the idea of moving the installation instructions to the main README. I do believe the likelihood that users run into problems here is reduced with recent changes to versioning and package loading.

One idea: Put the installation instructions only in the main README, and then in each notebook, have a small blurb of text that says something like: "Getting errors? See the README."